spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py

```diff
@@ -644,6 +665,9 @@ def is_recording(self):
     })
     def update_buffer(
             self, first_machine_time_step, run_until_timesteps):
+        """
+        UNUSED?
+        """
```

Comment:
Note the @inject statement - this is used if indirectly.

Reply:
fixed in my timing cherry pciks

Reply:
i made the buffer updator give the data. instead of it being injected. basically we have an extra api call for buffered input stuff. which avoids this

Reply:
OK, this is merged now - I updated the doc instead for now, but no probs if you have changed it elsewhere.